plugin/trino-redis/src/main/java/io/trino/plugin/redis/decoder/hash/HashRedisRowDecoder.java

```diff
@@ -62,4 +62,10 @@ public Optional<Map<DecoderColumnHandle, FieldValueProvider>> decodeRow(byte[] d
         }
         return Optional.of(decodedRow);
     }
+
+    @Override
+    public Optional<Map<DecoderColumnHandle, FieldValueProvider>> decodeRow(byte[] data)
+    {
+        return Optional.of(emptyMap());
```

Comment:
It would be better to throw an exception (e.g. `UnsupportedOperationException`) than dummy value if the method **_should not be_** called. Please add the test if this method **_should be_** called. Other decoders also.

Reply:
I changed it to throw an `UnsupportedOperationException` because the method should not be called.